cpp/include/cudf/lists/stream_compaction.hpp

```diff
@@ -82,17 +83,49 @@ std::unique_ptr<column> apply_boolean_mask(
  * @param input The input lists column
  * @param nulls_equal Flag to specify whether null elements should be considered as equal
  * @param nans_equal Flag to specify whether floating-point NaNs should be considered as equal
+ * @param keep_option Flag to specify which element to keep (first, last, any)
  * @param stream CUDA stream used for device memory operations and kernel launches
  * @param mr Device memory resource used to allocate the returned object
  * @return The resulting lists column containing lists without duplicates
  */
 std::unique_ptr<column> distinct(
+  // TODO: add defaults for nulls_equal, nans_equal, keep_option after distinct() is deprecated
```

Comment:
Can this comment be clarified?  `nulls_equal`, `nans_equal`, etc. already have defaults.  What needs to be added here?

Reply:
Actually, should we just remove this `TODO`?